<launch>
    <arg name="robot_ns" default="robot0"/>
    <node name="robot_node" type="robot_no_crash_node" ns="$(arg robot_ns)" pkg="robot_no_crash" output="screen"/>
    <node name="rqt" type="rqt_gui" pkg="rqt_gui"  />
</launch>